### PR TITLE
Drop prefixes, suffixes from error field members

### LINKED ISSUE #736: Drop `*Error` from error things ... and `payer*`
See: https://github.com/w3c/payment-request/pull/721#pullrequestreview-131715208

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #736

The following tasks have been completed:

 * [x] Confirmed there are no ReSpec errors/warnings (ignore the 2 warnings, unrelated!).
 * [ ] Added Web platform tests (link)
 * [ ] added MDN Docs (link)
 * [ ] added MDN [compat data](https://github.com/mdn/browser-compat-data)

Implementation commitment:

 * [ ] Safari (link to issue)
 * [ ] Chrome (link to issue)
 * [ ] Firefox (link to issue)
 * [ ] Edge (public signal)

Impact on Payment Handler spec?

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/pull/745.html" title="Last updated on Jul 5, 2018, 9:49 PM GMT (78fb2d1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/745/d27cc16...78fb2d1.html" title="Last updated on Jul 5, 2018, 9:49 PM GMT (78fb2d1)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@domenic, we need to keep `shippingAddressErrors` on `PaymentDetailsUpdate`, otherwise it would be weird... but chopped the rest:

```JS
dictionary PaymentDetailsUpdate : PaymentDetailsBase {
  DOMString error;
  PaymentItem total;
  AddressErrorFields shippingAddressErrors;
};
```

Gecko bug https://bugzilla.mozilla.org/show_bug.cgi?id=1473081

**Comment by marcoscaceres** [comment]
@domenic, hopefully quick review here. Implementations are underway in Blink and Firefox so if you could spare some cycles 🙏

File: index.html [hunk 1]
```diff
@@ -2795,18 +2795,18 @@ <h2>
...
-            DOMString dependentLocalityError;
-            DOMString languageCodeError;
-            DOMString organizationError;
-            DOMString phoneError;
-            DOMString postalCodeError;
-            DOMString recipientError;
-            DOMString regionError;
-            DOMString regionCodeError;
-            DOMString sortingCodeError;
+            DOMString addressLine;
```
Comment (domenic) [inline index.html:2798]: Unimportant nit: the order here is different than for AddressInit, which is itself different from PaymentAddress. *shrug*

**Review by domenic — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:2798]: Gonna go through the spec after all these parts land and put things in order. Right now just want to get things tested and implemented.

**Pull request merged**